azurerm_dns_zone_records_io_italia_it/variables.tf

```diff
@@ -0,0 +1,76 @@
+# Generic variables
+
+variable "environment" {
+  description = "The nick name identifying the type of environment (i.e. test, staging, production)."
+}
+
+variable "location" {
+  description = "The data center location where all resources will be put into."
+}
+
+variable "resource_name_prefix" {
+  description = "The prefix used to name all resources created."
+}
+
+# DNS module specific variables
+
+variable "azurerm_public_ip_name" {
```

Comment:
Description should be checked (seems a refuse)

Reply:
Seems correct to me. I've switched the order of name and suffix